src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -13,7 +13,7 @@ public class StorageConfiguration : ConfigurationCategory
         public PathSetting TempPath { get; set; }
 
         [Description("Use the 32 bits memory mapped pager, even when running in 64 bits")]
-        [DefaultValue(false)]
+        [DefaultValue(true)]
```

Comment:
Need to revert this

Reply:
This is draft and I want it to run on 32 bits (64 is by default)